pkg/router/httpTriggers.go

```diff
@@ -143,13 +144,21 @@ func versionHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func NotFoundHandler(w http.ResponseWriter, r *http.Request) {
+	log.Printf("404 Not Found: %s %s from %s at %s", r.Method, r.URL.Path, r.RemoteAddr, time.Now().Format(time.RFC3339))
+	w.WriteHeader(http.StatusNotFound)
+}
```

Comment:
This is going to be hard to parse in Axiom.

I suggest turning this into an anonymous function on line 160 and using the `ts.logger` so it's automatically serialized as json.

e.g.
```go
muxRouter.NotFoundHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
	ts.logger.Info("404 Not Found", zap.String("req_method", r.Method), /* etc... */)
	w.WriteHeader(http.StatusNotFound)
})
```

Try to re-use as many fields that as you can that we already index in Axiom.

e.g.
![CleanShot 2024-05-23 at 14 01 51@2x](https://github.com/gadget-inc/fission/assets/21965521/75521e06-c49e-4d61-ac2c-55ee8bf5fa37)